dcpquery/alembic/versions/e2b45add6f68_rename_bundle_and_file_tables.py

```diff
@@ -0,0 +1,52 @@
+"""rename_bundle_and_file_tables
+
+Revision ID: e2b45add6f68
+Revises: 7597ca0971e5
+Create Date: 2019-07-03 14:30:49.987853
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e2b45add6f68'
+down_revision = '7597ca0971e5'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.rename_table('bundles', 'bundles_all_versions')
+    op.rename_table('files', 'files_all_versions')
+    op.execute(
+        """
+        CREATE OR REPLACE RULE bundle_table_ignore_duplicate_inserts AS
+            ON INSERT TO bundles_all_versions
+                WHERE EXISTS (
+                  SELECT 1
+                FROM bundles_all_versions
+                WHERE fqid = NEW.fqid
+            )
+            DO INSTEAD NOTHING;
+        """
+    )
+    op.execute(
+        """
+        CREATE OR REPLACE RULE file_table_ignore_duplicate_inserts AS
+            ON INSERT TO files_all_versions
+                WHERE EXISTS (
+                  SELECT 1
+                FROM files_all_versions
+                WHERE fqid = NEW.fqid
+            )
+            DO INSTEAD NOTHING;
+        """
+    )
```

Comment:
I'm a little confused here-
- If rules attached to the old table automatically get reattached to the new renamed table, then why are new rules being inserted in the migration?
- If rules attached to the old table don't automatically get reattached, then should we clean up the old rules?

Reply:
I think that the rules get reattached to the renamed table, but we need to update the `FROM` statement to point to `files_all_versions` instead of files